core/server/storage/local-file-store.js

```diff
@@ -100,7 +100,19 @@ LocalFileStore.prototype.serve = function (options) {
         // CASE: serve images
         // For some reason send divides the max age number by 1000
         // Fallthrough: false ensures that if an image isn't found, it automatically 404s
-        return serveStatic(config.getContentPath('images'), {maxAge: utils.ONE_YEAR_MS, fallthrough: false});
+        return function serveStaticContent(req, res, next) {
+            return serveStatic(config.getContentPath('images'), {maxAge: utils.ONE_YEAR_MS, fallthrough: false})(req, res, function (err) {
+                if (err) {
+                    if (err.statusCode === 404) {
+                        return next(new errors.NotFoundError({message: i18n.t('errors.errors.pageNotFound')}));
+                    }
+
+                    return next(new errors.GhostError({err: err}));
+                }
+
+                next();
+            });
+        };
```

Comment:
I pushed this change as separate commit, see commit message for reasoning.
https://github.com/TryGhost/Ghost/pull/7431/commits/41747750253f2867a6d9d8f1a33e24391525bfd8

i hope that helps.